### ISSUE TITLE
set number of rows per page based on viewport height

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#329]
This would reduce a lot of up and down scrolling on mobile, while still showing large pages on larger screens.

https://github.com/react-tools/react-table/issues/552#issuecomment-336939953

**Comment by collinbarrett** [comment]
This should have a minimum page size greater than 0 for extra short viewports. Maybe 5 is a reasonable minimum?

**Comment by Efreak** [comment]
This should be optional. Some of us actually prefer having everything on one screen. Having said that, now that I know you're using viewport height, I can adjust for that (set window.innerHeight manually)

**Comment by collinbarrett** [comment]
@Efreak , thanks for that tip. Hmm. I'm open to that, for sure.

I just expect some browsers, especially on lower-powered devices, not to handle rendering over 2000 rows loaded at once.

But, I'm not a front-end specialist. So, I'm sure there's a way.
